docs/user-guide/migrating-to-1.0.0.md

```diff
@@ -111,8 +111,32 @@ In 0.x, the `cli` object was exported for use by external tools. It was later de
 
 ## Deprecating eslint-tester
```

Comment:
Well, `eslint-tester` is its own package that will still be available, so deprecating is the right term. You can still use it, but you do so at your own risk.